source/_integrations/ruuvi_gateway.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: Ruuvi Gateway
+description: Support for Ruuvi Gateway devices acting as BLE remote scanners.
+ha_category:
+  - Sensor
+ha_release: '2023.1'
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@akx'
+ha_domain: ruuvi_gateway
+ha_dhcp: true
+ha_integration_type: integration
+works_with:
+  - local
```

Comment:
```suggestion
```

Unless I missed something, I don't think this is official?

Reply:
I borrowed it from https://github.com/home-assistant/home-assistant.io/blob/11831492bf248b6f266b42ea55ed6ebc89397499/source/_integrations/esphome.markdown?plain=1#L32-L33

Reply:
Thats for the works with program. AFAICT Ruuvi hasn't joined that.
https://www.home-assistant.io/blog/2022/07/12/partner-program/

Reply:
Okay, thanks for the info. Removed!